packages/suite-desktop-core/e2e/support/testExtends/suiteBaseFixture.ts

```
@@ -0,0 +1,166 @@
+
+type StartEmuModelRequired = StartEmu & { model: Model };
+
+type suiteBaseFixture = {
+    startEmulator: boolean;
+    setupEmulator: boolean;
+    emulatorStartConf: StartEmuModelRequired;
+    emulatorSetupConf: SetupEmu;
+    url: string;
+    trezorUserEnvLink: TrezorUserEnvLinkClass;
+    page: Page;
+    exceptionLogger: void;
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)